plugins/bigquery/dbt/adapters/bigquery/impl.py

```diff
@@ -126,6 +77,16 @@ def _stub_relation(*args, **kwargs):
     )
 
 
+@dataclass
+class BigqueryConfig(AdapterConfig):
+    cluster_by: Optional[Union[List[str], str]] = None
+    partition_by: Optional[Dict[str, Any]] = None
+    kms_key_name: Optional[str] = None
+    labels: Optional[Dict[str, str]] = None
+    # TODO: should this accept `str` and `int`, too?
```

Comment:
Is this for `partitions`? I think a list of ints would also be valid for some use cases, but users can always pipe a list of ints to `| map('string')` or similar if needed

Reply:
Yeah. That sounds reasonable to me - I think it previously did, but I don't think it necessarily was intended to do so.